usr/share/rear/finalize/default/060_compare_files.sh

```diff
@@ -1,6 +1,6 @@
 if [ -e $VAR_DIR/layout/config/files.md5sum ] ; then
-    if ! chroot $TARGET_FS_ROOT md5sum -c --quiet < $VAR_DIR/layout/config/files.md5sum ; then
-        LogPrintError "Some configuration files in the restored system do not match the saved layout!"
+    if ! chroot $TARGET_FS_ROOT md5sum -c --quiet < $VAR_DIR/layout/config/files.md5sum 1>> >( tee -a "$RUNTIME_LOGFILE" 1>&7 ) 2>> >( tee -a "$RUNTIME_LOGFILE" 1>&8 ) ; then
+        LogPrintError "Error: configuration files shown above in the restored system do not match the saved layout!"
```

Comment:
I wonder if the message `... do not match the saved layout` is perhaps too specific
(is it really only about config files regarding the disk layout?) because
FILES_TO_PATCH_PATTERNS and CHECK_CONFIG_FILES
contain also bootloader config files and some other config files
like '/etc/rear/' and '/etc/udev/udev.conf'

Reply:
Addressed in 2f0b9374891831237c113a441ef8e7362718e151